src/jetstream/authnone.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"math"
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/labstack/echo"
+
+	"github.com/cloudfoundry-incubator/stratos/src/jetstream/repository/interfaces"
+)
+
+const (
+	noAuthUserID = "10000000-1111-2222-3333-444444444444"
+)
+
+//More fields will be moved into here as global portalProxy struct is phased out
+type noAuth struct {
+	databaseConnectionPool *sql.DB
+	localUserScope         string
+	consoleAdminScope      string
+	p                      *portalProxy
+}
+
+func (a *noAuth) ShowConfig(config *interfaces.ConsoleConfig) {
+	log.Info("... !!!!! No Authentication !!!!!")
+}
+
+//Login provides Local-auth specific Stratos login
+func (a *noAuth) Login(c echo.Context) error {
+	return errors.New("Can not login when there is no auth")
+}
+
+//Logout provides Local-auth specific Stratos login
+func (a *noAuth) Logout(c echo.Context) error {
+	return a.logout(c)
+}
+
+//GetUsername gets the user name for the specified local user
+func (a *noAuth) GetUsername(userid string) (string, error) {
+	return "admin", nil
```

Comment:
We should ensure there's no way to edit the user in the frontend. Think this should just be a case of apply a check for a scope?